addons/block_code/ui/picker/categories/category_factory.gd

```diff
@@ -192,3 +192,93 @@ static func add_to_categories(main: Array[BlockCategory], addition: Array[BlockC
 			main.append(add_category)
 
 	return main
+
+
+static func built_in_type_to_block_type(type: Variant.Type):
+	match type:
+		TYPE_BOOL:
+			return Types.BlockType.BOOL
+		TYPE_INT:
+			return Types.BlockType.INT
+		TYPE_FLOAT:
+			return Types.BlockType.FLOAT
+		TYPE_STRING:
+			return Types.BlockType.STRING
+		TYPE_VECTOR2:
+			return Types.BlockType.VECTOR2
+
+	return null
+
+
+static func property_to_blocklist(property: Dictionary) -> Array[Block]:
+	var block_list: Array[Block] = []
+
+	var block_type = built_in_type_to_block_type(property.type)
+
+	if block_type:
+		var type_string: String = Types.BlockType.find_key(block_type)
+
+		var b = BLOCKS["statement_block"].instantiate()
+		b.block_format = "Set %s {value: %s}" % [property.name, type_string]
```

Comment:
```suggestion
		b.block_format = "Set %s to {value: %s}" % [property.name, type_string]
```

Also it would be nice to [capitalize](https://docs.godotengine.org/en/stable/classes/class_string.html#class-string-method-capitalize) `property.name` like the Inspector does.